File: fizz-buzz.java
import java.util.*;

public class Main {
	private static String fizzbuzz(int i) {
		if (i % 3 == 0 && i % 5 == 0) return "Fizz Buzz";
		if (i % 3 == 0) return "Fizz";
		if (i % 5 == 0) return "Buzz";
		return String.valueOf(i);
	}

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int n = sc.nextInt();
		sc.close();
		for (int i = 1; i <= n; i++) System.out.println(fizzbuzz(i));
	}
}

File: fizz-buzz.kt
import java.util.*

fun fizzbuzz(i: Int): String {
    if (i % 3 == 0 && i % 5 == 0) return "Fizz Buzz"
    if (i % 3 == 0) return "Fizz"
    if (i % 5 == 0) return "Buzz"
    return i.toString()
}

fun main() {
    val sc = Scanner(System.`in`);
    val n = sc.nextInt()
    sc.close()
    for (i in 1..n) println(fizzbuzz(i))
}
